homeassistant/components/vacuum/__init__.py

```diff
@@ -168,9 +168,6 @@ def send_command(hass, command, params=None, entity_id=None):
 @asyncio.coroutine
 def async_setup(hass, config):
     """Set up the vacuum component."""
-    if not config[DOMAIN]:
```

Comment:
Yeah, I think I copied it from some other component, but I'm not sure why exactly, so better out...